### PR TITLE
GP2-261: additional media blocks on lesson

### PR DESCRIPTION
**Pull request by lewis-coulson-dit**
- [X] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-261
 - [X] Jira ticket has the correct status.
 - [X] [Changelog](CHANGELOG.md) entry added.

Adds the ability to add full-width video and image block in the body section of a lesson.

Screenshot 2020-10-12 at 08 31 45

### CODE REVIEW

**Review by bobmeredith — COMMENTED** [review]
Looks good! I've made a couple of very minor comments

File: learn/templates/learn/hero_media_item.html [hunk 1]
```diff
@@ -3,4 +3,4 @@
 {% for block in page.hero %}
     {% include_block block %}
 {% endfor %}
-<div id="lesson-hero-video-transcript"></div>
\ No newline at end of file
+<div class="video-transcript-container"></div>
```
[the remark this answers is not in the document]
Comment (lewis-coulson-dit) [reply learn/templates/learn/hero_media_item.html]: Added to video tag template

File: core/templates/core/includes/_video_full_width.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% load video_tags %}
+
+<div class="media-section m-v-s">
+    {% render_video block.value %}
+    <div class="video-transcript-container"></div>
+</div>
```
[the remark this answers is not in the document]
Comment (lewis-coulson-dit) [reply core/templates/core/includes/_video_full_width.html:5]: Can't see an if here

**Review by bobmeredith — APPROVED** [review]
LGTM

**Pull request merged**